packages/dart_firebase_admin/lib/src/app/firebase_admin.dart

```diff
@@ -24,8 +24,17 @@ class FirebaseAdminApp {
   /// Use the Firebase Emulator Suite to run the app locally.
   void useEmulator() {
     _isUsingEmulator = true;
-    authApiHost = Uri.http('127.0.0.1:9099', 'identitytoolkit.googleapis.com/');
-    firestoreApiHost = Uri.http('127.0.0.1:8080', '/');
+    final env =
+        Zone.current[envSymbol] as Map<String, String>? ?? Platform.environment;
```

Comment:
If you're going to use zones, have your test use it :)

Reply:
Completely missed this, thanks :). Resolved now.